src/mbgl/renderer/painters/painter_symbol.cpp

```diff
@@ -68,13 +68,11 @@ void Painter::renderSymbol(PaintParameters& parameters,
         auto paintPropertyValues = layer.iconPaintProperties();
 
         SpriteAtlas& atlas = *bucket.spriteAtlas;
-        const bool iconScaled = layout.get<IconSize>().constantOr(1.0) != 1.0 ||
-            frame.pixelRatio != atlas.getPixelRatio() ||
```

Comment:
Is this condition now being covered [here](https://github.com/mapbox/mapbox-gl-native/pull/9038/commits/67d5254f71fb643036dcf45674fb477bceed219b#diff-f27a3a157c549a13f8fe1ee8c8157ea8R274)?